sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -1685,76 +1685,6 @@ TEST_F(CommandGraphTest, BindlessExceptionCheck) {
   sycl::free(ImgMemUSM, Ctxt);
 }
 
-TEST_F(CommandGraphTest, GetProfilingInfoExceptionCheck) {
```

Comment:
Does it make sense to rephrase that into a unittest checking that the events are set and freed correctly?

Reply:
I'm not sure I understand what you would like to test.
The `event_status_querying.cpp` test already verifies that events states are correctly set.

Reply:
I was wondering if we can test internally with an unittest whether the start event is correctly set, used (e.g., if it is taken from the first command-list), and freed. I think this could make sense since we extend the typical profiling path.

Reply:
This type of test could be relevant but unitests do not use actual backends but only a plug-in mock. So we can't test that in an unitest.
And (assuming we can re-read a level-zero commandlist, which I'm not sure we can do) I don't see this type of tests consistent with the purpose of e2e tests.

Reply:
True. I missed that this is only L0. Please disregard my comments then.